src/Uno.AzureDevOps/Uno.AzureDevOps.Droid/Uno.AzureDevOps.Droid.csproj

```diff
@@ -52,13 +52,13 @@
     <AndroidCreatePackagePerAbi>true</AndroidCreatePackagePerAbi>
     <EmbedAssembliesIntoApk>true</EmbedAssembliesIntoApk>
     <AotAssemblies>true</AotAssemblies>
+		<AndroidAotAdditionalArguments>nodebug,no-write-symbols</AndroidAotAdditionalArguments>
     <EnableLLVM>false</EnableLLVM>
     <AndroidEnableSGenConcurrent>true</AndroidEnableSGenConcurrent>
     <LangVersion>7.1</LangVersion>
     <BundleAssemblies>false</BundleAssemblies>
     <UseHighDPIResources>true</UseHighDPIResources>
-    <AndroidLinkSkip>
-    </AndroidLinkSkip>
+		<AndroidLinkSkip>$(AssemblyName)</AndroidLinkSkip>
```

Comment:
That's weird to change the _Android_ project to fix a problem on _iOS_.

Reply:
Put in in this PR as it has been requested (to avoid two PRs ...)